xtask/src/qemu.rs

```diff
@@ -506,6 +506,23 @@ pub fn run_qemu(arch: UefiArch, opt: &QemuOpt) -> Result<()> {
         None
     };
 
+    // Make PCI tree a little more complicated so the PCI enumeration integration
```

Comment:
Could you perhaps create a brief ASCII art in the comment describing the PCIe topology, please?